manifests/bucketeer/charts/web/values.yaml

```diff
@@ -1,24 +1,112 @@
-fullnameOverride: "web-gateway"
+image:
+  repository: ghcr.io/bucketeer-io/bucketeer-web
+  pullPolicy: IfNotPresent
+fullnameOverride: "web"
 namespace: default
 env:
+  cloudService:
+  profile: true
+  bucketeerTestEnabled:
   gcpEnabled: true
+  bigqueryEmulatorHost:
+  pubsubEmulatorHost:
+  project:
+  mysqlUser:
+  mysqlPass:
+  mysqlHost:
+  mysqlPort: 3306
+  mysqlDbName:
```

Comment:
I couldn't find the `mysqlDbOpenConns` value.

Reply:
We only added `mysqlDbOpenConns` for subscribers before, we can add it here if needed.